src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Ruby agent 8.x to 7.x migration guide
+tags:
+  - Agents
+  - Ruby agent
+  - Getting started
+redirects:
+  - /docs/agents/ruby-agent/getting-started/migration-9x-guide
+---
+
+## Summary [#summary]
+
+This guide covers the major changes between the 8.x and 9.x series of the Ruby agent, issues that may be encountered while upgrading, and how to successfully migrate to version 9.x.
+
+The main changes include:
+
+* [Removed all deprecated configuration options](#removed_config)
+* [Dropped support for Ruby 2.2 and 2.3](#ruby_2223)
+* [Enabled Thread tracing](#thread_tracing)
+* [Removed deprecated instrumntation for gems](#removed_gem_instrumentation)
+* [Updated API method `set_transaction_name`](#set_transaction_name)
+* [Removed deprecated API methods](#removed_api_methods)
+
+See the [milestone for 9.0](https://github.com/newrelic/newrelic-ruby-agent/milestone/28?closed=1) for more information.
+
+## Removed Deprecated Configuration [#removed_config]
+
+  The following configuration options have been removed and will no longer work. Please update all configs to use the replacements listed below. Please see our [configuration documentation](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration) for additional details.
+
+  |  Removed                                  | Replacement                               | `newrelic.yml` example                                                              |
```

Comment:
Out of curiosity, does the `newrelic.yml` example have any pattern to the content? (ex. all default values, all values that disable the instrumentation)

Reply:
@kaylareopelle, @tannalynn might know this better at the top of her head, but I can't discern a common pattern.